GetUserData - adding an 'attributes'  parameter (#37908)
demisto/content: Packs/CommonScripts/ReleaseNotes/1_19_1.md

#### Scripts

##### get-user-data

> - Added support for getting more information about a user when running ***ad-get-user*** command via this script by adding the **attributes** parameter.

Reviewer: ```suggestion
- Added the **attributes** parameter to the ***ad-get-user*** command in this script to enabling the retrieval of additional user information.
```
